File: Javascript/Ajax/csharp/usercontrols/rss.ascx.cs
using System;
using System.Collections;
using System.Configuration;
using System.Data;
using System.Web;
using System.Web.Security;
using System.Web.UI;
using System.Web.UI.HtmlControls;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;

public partial class usercontrols_rss : System.Web.UI.UserControl
{
    public string Title
    {
        get
        {
            return lblTitle.Text;
        }
        set
        {
            lblTitle.Text = value;
        }
    }

    public string Description
    {
        get
        {
            return lblDescription.Text;
        }
        set
        {
            lblDescription.Text = value;
        }
    }

    public string Link
    {
        get
        {            
            return hlRss.NavigateUrl;
        }
        set
        {
            hlRss.NavigateUrl = value;
        }
    }

    public object DataSource
    {
        set
        {
            dlRSS.DataSource = value;
            dlRSS.DataBind();
        }
    }
      

    protected void Page_Load(object sender, EventArgs e)
    {

    }
}

File: Javascript/Ajax/vb.net/usercontrols/rss.ascx.vb

Partial Class usercontrols_rss
    Inherits System.Web.UI.UserControl

    Public Property title() As String
        Get
            Return lblTitle.Text
        End Get
        Set(ByVal value As String)
            lblTitle.Text = value
        End Set
    End Property

    Public Property description() As String
        Get
            Return lblDescription.Text
        End Get
        Set(ByVal value As String)
            lblDescription.Text = value
        End Set
    End Property

    Public Property link() As String
        Get
            Return hlRss.NavigateUrl
        End Get
        Set(ByVal value As String)
            hlRss.NavigateUrl = value
        End Set
    End Property

    Public WriteOnly Property DataSource() As Object
        Set(ByVal value As Object)
            dlRSS.DataSource = value
            dlRSS.DataBind()
        End Set
    End Property
End Class
